modules/ROOT/pages/_partials/link-anypoint-platform-accounts.adoc

```diff
@@ -0,0 +1,83 @@
+// tag::profileLinkingContextForUsers[]
+[[profile-linking-context-users]]
+= Linking Your Anypoint Platform Profiles
+
+When you log in to Anypoint Platform for the first time using an external identity provider (IdP), Anypoint Platform locates potential user accounts matched by your email address that is associated with the IdP profile. You are prompted to link your IdP profile with the existing accounts. Your options for linking profiles vary based on whether the profiles use the Anypoint identity provider (IdP) or an external IdP that uses single sign-on (SSO). Anypoint Platform supports:
+
+* Linking an SSO login to a local profile by using credentials (login managed by Anypoint IdP)
+* Linking an SSO login to another SSO login that uses a different IdP to log in
+* Linking an SSO login to multiple other profiles that each use different IdPs to log in
+
+Anypoint Platform does not support linking multiple profiles that use the same IdP. The IdP must be an OpenID Connect (OIDC) provider. You can link up to 25 user profiles. This feature is available only if an xref:external-identity-index.adoc#enable-profile-linking[organization administrator has enabled it].
+
+// end::profileLinkingContextForUsers[]
+
+// tag::profileLinking[]
+[[profile-linking]]
+= Linking an existing SSO Login to Additional Logins
+
+When you attempt to login using SSO and there are additional user profiles that share the same email address but use different identity providers (IdPs) to log in, you are prompted to link the profiles. Note that you cannot link profiles that use the same IdP.
+
+. In the Anypoint Platform login page, click *Sign in with <your identity provider>*.
+. If prompted, enter your credentials for your IdP. +
+If your SSO credentials share an email address with additional profiles in the same organization, you are prompted to select one of the profiles and log in with those credentials to link your profiles. If you are not prompted, your other profiles use a different email address than the SSO login you are logging in to.
```

Comment:
```suggestion
If your SSO credentials share an email address with additional profiles in the same organization, you are prompted to select one of the profiles and log in with those credentials to link your profiles. If you are not prompted, your other profiles use a different email address than the SSO login that you are logging in to.
```

Reply:
Do you need the last sentence? I think its meaning is implied, but up to you.

Reply:
```suggestion
If your SSO credentials share an email address with additional profiles in the same organization, you are prompted to select one of the profiles and log in with those credentials to link your profiles. 
```